config/config.go

```diff
@@ -537,33 +537,53 @@ func getBootstrapConfig(v *viper.Viper, networkID uint32) (node.BootstrapConfig,
 		return node.BootstrapConfig{}, fmt.Errorf("set %q but didn't set %q", BootstrapIDsKey, BootstrapIPsKey)
 	}
 
-	bootstrapIPs, bootstrapIDs := genesis.SampleBeacons(networkID, 5)
+	bootstrappers := genesis.SampleBootstrappers(networkID, 5)
```

Comment:
This code is really pretty jank and confusing. Should we replace `config.BootstrapIPs` and `config.BootstrapIDs` with `config.Bootstrapers`?

Reply:
+1000, I think combining them into a single flag value of JSON format is much simpler.